handler_error.go

```diff
@@ -87,6 +88,14 @@ func (e *ErrorHandler) HandleError(w http.ResponseWriter, r *http.Request, errMs
 
 		t := time.Now()
 
+		if ctxGetDefaultVersion(r) {
+			if vinfo := ctxGetVersionInfo(r); vinfo != nil {
+				if config.Global.DefaultVersionHeader {
+					w.Header().Set(defaultVersionHeader, vinfo.Name)
```

Comment:
I assume you need to remove `defaultVersionHeader` constant and replace it with `config.Global.DefaultVersionHeader` in all places.